modules/mobile/spec/models/adapters/appointments_vaos_v2_adapter_spec.rb

```diff
@@ -688,4 +701,67 @@
       expect(no_parentheses_phone_num_appt.first[:patient_phone_number]).to eq('480-293-1922')
     end
   end
+
+  describe 'embedded acheron values' do
+    let(:acheron_appointment) { adapted_appointment[12] }
+
+    # these tests are duplicative of the full body test but are meant to highlight the relevant data
+    it 'parses values out of the reason code' do
+      expect(acheron_appointment.patient_email).to eq('melissa.gra@va.gov')
+      expect(acheron_appointment.patient_phone_number).to eq('317-448-5062')
+      expect(acheron_appointment.proposed_times).to eq([{ date: '12/13/2022', time: 'PM' },
+                                                        { date: '12/21/2022', time: 'AM' }])
+      expect(acheron_appointment.comment).to eq('My leg!')
+      expect(acheron_appointment.reason).to eq('ROUTINEVISIT')
```

Comment:
I'm a bit confused on the reason text conversion. ticket 4267 was already closed but it doesn't look like the conversion is taking place. I was expecting the all caps text here to convert over to something like "This is a routine or follow up visit" or however the wording was before v2.

Reply:
ah, didn't read far enough to realize that was just a discovery ticket. sounds good!